ext/pcre/php_pcre.c

```diff
@@ -49,9 +49,9 @@ char *php_pcre_version;
 
 struct _pcre_cache_entry {
 	pcre2_code *re;
+	zend_string **subpats_table;
```

Comment:
Maybe document that the list is null terminated? Or is there a way to know how many elements are in the table?

Reply:
Both. The size is just equal to capture_count+1 but I can document that. (Note that this was already the case)